pages/program/program.md

```diff
@@ -1281,14 +1280,14 @@ tags will be replaced.-->
   <td rowspan=2 height=290 class=xl736507 width=108 style='height:217.5pt;
   width:81pt'>10:30 AM</td>
   <td colspan=2 class=xl786507 width=302 style='width:227pt'><font
-  class="font116507">Session 1A</font><font class="font66507">: <a href="{{ site.baseurl }}/program/sessions/#session-1a">RSE Pedagogy</a>, </font><font
+  class="font116507">Session 1A</font><font class="font66507">: <a href="{{ site.baseurl }}/program/sessions/#session-1a">RSE Pedagogy</a>, </font>Charles Ferenbaugh, <font
   class="font106507">Hopi/Tewa</font></td>
-  <td colspan=2 class=xl1066507 width=284 style='border-left:none;width:213pt'>Session 1B<font class="font56507">:
-  <a href="{{ site.baseurl }}/program/sessions/#session-1b">AI, ML, and Automation</a>, </font><font
+  <td colspan=2 class=xl1066507 width=284 style='border-left:none;width:213pt'><font
+  class="font116507">Session 1B</font><font class="font66507">: <a href="{{ site.baseurl }}/program/sessions/#session-1b">AI, ML, and Automation</a>, </font>Patrick Diehl, <font
```

Comment:
@mrmundt Halp. Y bold? https://output.circle-artifacts.com/output/job/e1489516-fcb0-4f0f-b93f-973fd377f14e/artifacts/0/usrse.github.io/usrse24/program/index.html

Reply:
Well isn't that a pickle... I have no idea. Try wrapping it in:
```
<font class="font116507"></font>
```